Makefile

```diff
@@ -27,4 +27,7 @@ ch_tunnel: ## Create tunnel connection to Clickhouse eg. make ch_tunnel zone eur
 	@echo "Connecting to Clickhouse"
 	@gcloud compute ssh ${instance} --zone="${zone}" --tunnel-through-iap -- -L 8123:localhost:8123
 
-.PHONY: help run_local debug_local es_tunnel ch_tunnel
\ No newline at end of file
+run_with_standalone: ## Runs API with standalone platform
+	@sbt "run 8090" -DELASTICSEARCH_HOST=elasticsearch -DSLICK_CLICKHOUSE_URL=jdbc:clickhouse://clickhouse:8123
```

Comment:
I may be wrong, but in this case, having the standalone deployment running, as we'd be running the API on the host OS, but elastic search and click house via docker-compose, in fact, to the eyes of the API, those services are on _localhost_.

Reply:
Ideally, we'd like for those values to be _localhost_ by default, so we can launch the API in the IDE straight away, to quickly test changes in the code.